lib/cli-sb/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "sb",
+  "version": "5.2.0-rc.9",
```

Comment:
wrong version?

Reply:
Will get boosted to `5.3.0-alpha.0` when I do the release

Reply:
I mean, it's rc-11 in other packages